docs/core/install/linux-package-mixup.md

```diff
@@ -0,0 +1,136 @@
+  
+    For Fedora, CentOS 8+, RHEL 8+, use the following bash commands:
+
+    ```bash
+    echo 'priority=50' | sudo tee -a /etc/yum.repos.d/microsoft-prod.repo
+    sudo dnf remove 'dotnet*' 'aspnet*' 'netstandard*'
+    sudo dnf install dotnet-sdk-5.0
+    ```
+
+04. **I've encountered a bug in the Linux distribution version of .NET, I need the latest Microsoft version.**
+
+    Use solution 3 to solve this problem.
+
+## Online references
+
+Many of these problems have been reported by users such as yourself. The following is a list of those issues. You can read through them for insights on what may be happening:
+
+- System.IO.FileNotFoundException and '/usr/share/dotnet/packs/Microsoft.NETCore.App.Ref/5.0.0/data/FrameworkList.xml'
+
+  - [SDK #15785: unable to build brand new project after upgrading to 5.0.3](https://github.com/dotnet/sdk/issues/15785)
+  - [SDK #15863: "MSB4018 ResolveTargetingPackAssets task failed unexpectedly" after updating to 5.0.103](https://github.com/dotnet/sdk/issues/15863)
+  - [SDK #17411: dotnet build always throwing error](https://github.com/dotnet/sdk/issues/17411)
+  - [SDK #12075: dotnet 3.1.301 on Fedora 32 unable to find FrameworkList.xml because it doesn't exist](https://github.com/dotnet/sdk/issues/12075)
+
+- Fatal error: _libhostfxr.so_ couldn't be found
+
+  - [SDK #17570: After updating Fedora 33 to 34 and dotnet 5.0.5 to 5.0.6 I get error regarding libhostfxr.so](https://github.com/dotnet/sdk/issues/17570):
+
+- Fatal error: folder _/host/fxr_ doesn't exist
+
+  - [Core #5746: The folder does not exist when installing 3.1 on CentOS 8 with packages.microsoft.com repo enabled](https://github.com/dotnet/core/issues/5746)
+  - [SDK #15476: A fatal error occurred. The folder [/usr/share/dotnet/host/fxr] does not exist](https://github.com/dotnet/sdk/issues/15476):
+
+- Fatal error: folder _/host/fxr_ doesn't contain any version-numbered child folders
+
+  - [Installer #9254: Error when install dotnet/core/aspnet:3.1 on CentOS 8 - Folder does not contain any version-numbered child folders](https://github.com/dotnet/installer/issues/9254)
+  - [StackOverflow: Error when install dotnet/core/aspnet:3.1 on CentOS 8 - Folder does not contain any version-numbered child folders](https://stackoverflow.com/questions/65422998/)
+
+- Generic errors without clear messages
+
+  - [Core #4605: cannot run "dotnet new console"](https://github.com/dotnet/core/issues/4605)
+  - [Core #4644: Cannot install .NET Core SDK 2.1 on Fedora 32](https://github.com/dotnet/core/issues/4655)
+  - [Runtime #49375: After updating to 5.0.200-1 using package manager, it appears that no sdks are installed](https://github.com/dotnet/runtime/issues/49375)
+
+## Next steps
+
+- [Install .NET on Linux](linux.md)
```

Comment:
Heh, maybe installing .NET isn't the next step after troubleshooting the .NET installation? :smile:

Reply:
It's the "overview" and list of supported versions. I should probably change the title here...